bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/BallerinaErrors.java

```diff
@@ -61,6 +61,7 @@
     public static final String GENERATE_PKG_START = "___start_";
     public static final String GENERATE_PKG_STOP = "___stop_";
     public static final String GENERATE_OBJECT_CLASS_PREFIX = ".$value$";
+    public static final String ASYNC_CALL_INSIDE_LOCK = "AsyncCallInsideLock";
```

Comment:
Shall we move this constant to BallerinaErrorReasons.java and prefix it with "{ballerina}"? (Check BallerinaErrorReasons for examples).

And I guess we should suffix it with "Error"?

```suggestion
    public static final String ASYNC_CALL_INSIDE_LOCK = "AsyncCallInsideLockError";
```